### PR TITLE
chore: deploying 2021.07 to QA_NCT

### PR DESCRIPTION
**Pull request by atharvar28**
Deploying `2021.07` release version to QA_NCT

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# qa-niaid.planx-pla.net/manifest.json
## Deployment changes
- arborist
  - Needs Fence later than 4.22.2 or 2020.08 (which introduces the new JWT scopes claim). (uc-cdis/uc-cdis/arborist#130)
- indexd
  - When running gen3authz,  backoff package needed separate poetry run pip install (uc-cdis/uc-cdis/indexd#304)

File: qa-niaid.planx-pla.net/manifest.json [hunk 1]
```diff
@@ -7,27 +7,27 @@
     "autodeploy": "yes"
   },
   "versions": {
-    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2021.04",
+    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2021.07",
     "audit-service": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/audit-service:1.0.0",
```
Comment (paulineribeyre) [inline qa-niaid.planx-pla.net/manifest.json:11]: why not audit-service too?

**Comment by PlanXCyborg** [comment]
# qa-niaid.planx-pla.net/manifest.json
## Deployment changes
- arborist
  - Needs Fence later than 4.22.2 or 2020.08 (which introduces the new JWT scopes claim). (uc-cdis/uc-cdis/arborist#130)
- indexd
  - When running gen3authz,  backoff package needed separate poetry run pip install (uc-cdis/uc-cdis/indexd#304)

File: qa-niaid.planx-pla.net/manifest.json (hunk 1 above)
Comment (atharvar28) [reply qa-niaid.planx-pla.net/manifest.json]: Yes I wanted to ask couple of questions latest version of fence and make changes to both accordingly. Thank you for clarifying it on slack

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**